guide/additional-info/async-await.md

```diff
@@ -0,0 +1,163 @@
+# Understanding async/await
+
+If you aren't very familiar with ECMAScript 2017, you may not know about async/await. It's a useful way to handle Promises in a synchronous manner instead stacking `.then()` callbacks. Also, it looks cleaner and is increases readability overall.
+
+## How do Promises work?
+
+Before we can get into async/await, you should know what Promises are and how they work, because async/await is just a way to handle Promises. if you know "what Promises are and how to deal with them" you can skip this part. 
```

Comment:
Second sentence doesn't need the quotes, and the beginning of the sentence needs to be capitalized. A comma before "you can skip this part" would make it less awkward to read, I think.